### ISSUE TITLE
Handle multi languages plugin

### ISSUE THREAD

**Issue by bidoubiwa** [meilisearch/strapi-plugin-meilisearch#85]
## Problem

For now the plugin only works in English as the text has been hard-coded inside the different components.

## Solution

Strapi provided a very good way to handle multiple languages. 
It has been tried in the header of the MeiliSearch plugin `meilisearch/admin/src/containers/HomePage/index.js`

```js
<Header
            title={{
              label: formatMessage({ id: getTrad('plugin.name') })
            }}
            content={formatMessage({ id: getTrad('header.description') })}
          />
```

The text is imported from `meilisearch/admin/src/translations`. 

We should change all `hard-coded` text with the same feature.

### Example

```
'Indexed In MeiliSearch'
```

should become
```
// collections.js
formatMessage({ id: getTrad('collection.indexed') })
```

And the following should be added in at least `en`
```js
//meilisearch/admin/src/translations
"collection.indexed": "Indexed in MeiliSearch"
```

## Whats next

Once this is done, it will be very easy to add new languages to the plugin

**Comment by abhijeet007rocks8** [comment]
@curquiza Is there list of files/file on which refactoring needs to be done?
Also can you explain the last part in issue description like where should that be added ?

**Comment by curquiza** [comment]
Hello @abhijeet007rocks8, I let @bidoubiwa explains this since it's her issue and she's the maintainer of the repo 🙂

**Comment by bidoubiwa** [comment]
hey @abhijeet007rocks8 ! I assign you to this issue :)  If you want to find out how to do the translation system I advise to look et this plugin that implemented it: 
https://github.com/alexzaganelli/strapi-plugin-email-designer/tree/main/admin/src/translations

Example: 
```js
           />
            <Button onClick={saveDesign} color="success">
              {formatMessage({ id: getTrad('designer.action.saveTemplate') })}
            </Button>

```

**Comment by bidoubiwa** [comment]
Hello @abhijeet007rocks8  I have removed the assignement as we have no news from you! Feel free to open the PR anyway.

**Comment by mimartinez** [comment]
Hi, I have opened a new PR with the development for the plugin to handle multiple languages. (`en`,`es`)

![Captura de pantalla 2024-04-04 a las 2 54 19](https://github.com/meilisearch/strapi-plugin-meilisearch/assets/888965/4be84cde-e8f7-46d3-adec-3c6258d2211b)
![Captura de pantalla 2024-04-04 a las 2 50 12](https://github.com/meilisearch/strapi-plugin-meilisearch/assets/888965/da4b8961-fae6-48d2-b1d3-13d3bf02b6eb)
![Captura de pantalla 2024-04-04 a las 2 54 37](https://github.com/meilisearch/strapi-plugin-meilisearch/assets/888965/e1467b3b-b3b1-46f1-bfa7-cdfbe71336c4)
